bucket/carotdav.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://rei.to/carotdav_en.html",
+    "description": "A simple WebDAV / FTP / SFTP / Online Storages client for Windows OS",
+    "license": {
+        "url": "http://rei.to/carotdav_en.html#license"
+    },
+    "version": "1.15.5",
+    "url": "http://rei.to/CarotDAV1.15.5.portable.zip",
+    "hash": "21b0b68ec45223c747d67b0e21ed87e8c195d52e5fc651b30b631a2f328bcd01",
+    "extract_dir": "CarotDAV",
+    "bin": [
+        "CarotDAV.exe",
+        "CarotDAVC.exe"
+    ],
+    "persist": "Setting.xml",
+    "pre_install": "if(!(Test-Path \"$persist_dir\\Setting.xml\")) {Set-Content -Path (\"$dir\\Setting.xml\") -Value ('<?xml version=\"1.0\" encoding=\"utf-8\"?><WebDAVClientSetting xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\"></WebDAVClientSetting>')}",
+    "checkver": {
+        "url": "http://rei.to/carotdav.version.txt",
+        "regex": "([\\d\\.]+)"
+    },
+    "autoupdate": {
+        "url": "http://rei.to/CarotDAV$version.portable.zip"
```

Comment:
Add hash extraction
```suggestion
        "url": "http://rei.to/CarotDAV$version.portable.zip",
        "hash": {
            "url": "http://rei.to/carotdav_en.html#download",
            "regex": "SHA-1: $sha1"
        }
```

Reply:
The first matched hash is sha1 of installer version. 
I can't found the way to capture second matched hash which belongs to portable version.
It could be nice if `hash` has the `reverse: true` option.

Reply:
`$basename">Ver. $version SHA-1: $sha1` Should do the trick

Reply:
Edited suggestion to reflect the change

Reply:
![image](https://user-images.githubusercontent.com/13260377/54995779-e32d6500-4fc7-11e9-97cb-ac14f79672fc.png)

Reply:
Nice job!